setup.py

```diff
@@ -194,4 +194,14 @@ def find_any_distribution(pkgs):
 chainerx_build_helper.config_setup_kwargs(setup_kwargs, build_chainerx)
 
 
+def emit_build_info():
+    filename = 'chainerx/_build_info.py'
+    build_info = 'build_chainerx = {}\n'.format(build_chainerx)
+    with open(filename, mode='w') as f:
+        f.write(build_info)
+
+
+emit_build_info()
```

Comment:
* These code should be written in `chainerx_build_helper.py`.
*  Don't call it in module-level code. Perhaps `CMakeBuild.build_extension`.
* Do not assume the current directory.

Reply:
I'll move it to the top of `config_setup_kwargs` because `_build_info.py` should be emitted even if ChainerX is not built.

Reply:
>Do not assume the current directory.

This means you should not use the current directory. That's because `setup.py` can be called from anywhere. It might be ok if it's called within `CMakeBuild.build_extension`. If my memory is correct setuptools changes the current directory appropriately.

>I'll move it to the top of `config_setup_kwargs` because `_build_info.py` should be emitted even if ChainerX is not built.

Probably you should make another extension to be used in no-chainerx build.